lib/helpers/slide_route.dart

```diff
@@ -102,4 +104,5 @@ class SlideBottomRoute extends PageRouteBuilder<SlideBottomRoute> {
           ),
         );
   final Widget page;
+  final RouteSettings settings;
```

Comment:
As mentioned previously don't create this settings property in this class, you can directly pass the object from route_page.dart